processor/Cargo.toml

```diff
@@ -19,3 +19,11 @@ authors = ["Bitwise IO"]
 
 [dependencies]
 bond-common = { path = "../common"}
+protobuf = "2"
+rust-crypto = "0.2.36"
+zmq = { git = "https://github.com/erickt/rust-zmq", branch = "release/v0.8" }
```

Comment:
Do you get any issues if you omit this dependency? The crate is not being directly used in Bond, so I don't think you need it here. It is used by the Sawtooth SDK, which is why you need to `apt-get install` the `libzmq3-dev` library in the container (that library is a prereq for this zmq wrapper crate).